user-guide/Advanced_Functionality/Swarming/EnableSwarming.md

```diff
@@ -47,6 +47,15 @@ To run a prerequisites check using SLNetClientTest tool:
 
    For each prerequisite, the tool will indicate whether the prerequisite is met (*True*) or not (*False*). If you hover over the *Summary* item, you will get a detailed overview of which items cause prerequisites not to be met (e.g. specific scripts, enhanced service connectors, connectors using the obsolete alarm ID format, etc.).
 
+> [!NOTE]
+> Obsolete Engine methods are only included in the prerequisites check from DataMiner 10.5.3 onwards<!--RN 42073-->. If you are using DataMiner 10.5.1 or 10.5.2, these obsolete methods may still be present even if the prerequisite check does not report any issues.
+> The following Engine methods are obsolete and should not be used:
+>
+> - GetAlarmProperty
```

Comment:
please add the arguments to this list as these methods are replaced by methods with the same name but different arguments.

E.g.

GetAlarmProperty(int, int, string) is now obsolete and replaced by GetAlarmProperty(AlarmTreeID, string)


full list:

GetAlarmProperty(int, int, string)
SetAlarmProperty(int, int, string, string)
SetAlarmProperties(int, int, string[], string[])
AcknowledgeAlarm(int, int, string)